### ISSUE TITLE
Add information about customCardProps to Detailed Syntax Reference section

### ISSUE THREAD

**Issue by balktl** [SharePoint/sp-dev-docs#6641]
On **https://docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/column-formatting** - could customCardProps be added into the detailed syntax section? There's a sample for it in the page (https://docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/column-formatting#custom-cards-on-hover), but I don't know what other accepted values are available for this property.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f60068c3-4534-a400-350b-fdbcefd684b6
* Version Independent ID: 228accdb-a7de-0671-34ad-60a37866c380
* Content: [Use column formatting to customize SharePoint](https://docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/column-formatting)
* Content Source: [docs/declarative-customization/column-formatting.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/declarative-customization/column-formatting.md)
* Product: **sharepoint**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by bcameron1231** [comment]
Hi, can you clarify a bit? Is your question concerning whether/if there are other properties available aside from openOnEvent, directionHint, isBeakVisible and beakStyle? Or what values are available for each of the properties?

**Comment by balktl** [comment]
@bcameron1231 - Correct, what do those properties control and are there additional values available for each of the properties. When reading this documentation I was thinking it would be covered in the Detailed syntax reference section, but it's not there.

**Comment by bcameron1231** [comment]
Hi Ah I see what you mean. If you open up JSON schema which is used by column formatting, you will see what options are available. https://developer.microsoft.com/json-schemas/sp/v2/column-formatting.schema.json

I have listed them below (options are comma seperated)

**openOnEvent** - click, hover
**directionalHint** - topCenter, bottomCenter, leftCenter, rightCenter
**isBeakVisible** - true, false
**beakStyle** - any css properties you want.

**Comment by balktl** [comment]
Yes, that's what I was looking for. I didn't think to look in the schema definition. https://developer.microsoft.com/json-schemas/sp/v2/column-formatting.schema.json is peppered throughout the documentation and I didn't think about navigating there.
